src/app/components/SelectedCoinCharts/index.tsx

```diff
@@ -607,14 +606,14 @@ export default function SelectedCoinsCharts() {
                       <div className="w-4 h-4 rounded-sm bg-[#7878FA] text-xl"></div>
                       <p className="text-[#424286] dark:text-[#D1D1D1] lg:text-sm">
                         {coin1?.name} {handleCurrencySymbol(currentCurrency)}
-                        {handleCurrency(coin1?.total_volume)}
+                        {handleCurrency(coin1?.total_volume[currentCurrency])}
                       </p>
                     </div>
                     <div className={"flex items-center space-x-2"}>
                       <div className="w-4 h-4 rounded-sm bg-[#9D62D9] text-xl"></div>
                       <p className="text-[#424286] dark:text-[#D1D1D1] lg:text-sm">
                         {coin2?.name} {handleCurrencySymbol(currentCurrency)}
-                        {handleCurrency(coin2?.total_volume)}
+                        {handleCurrency(coin2?.total_volume[currentCurrency])}
```

Comment:
a better name for this is format, handle is used for event listener callbacks, no need to change tho